src/entity/Car.java

```diff
@@ -0,0 +1,65 @@
+package entity;
+
+import car_action.CarActions;
```

Comment:
the name of packages must not contain _ symbols. The package `entity `should be renamed as well because `entity `more applicable web applications, here it is just a model.